.github/workflows/ci.yml

```diff
@@ -54,9 +54,9 @@ jobs:
       - name: Test FSharp.Common.Tests
         working-directory: ./ReSharper.FSharp
         run: dotnet test test/src/FSharp.Common.Tests/FSharp.Common.Tests.fsproj --logger:nunit --results-directory TestResults/FSharp.Common.Tests
+        continue-on-error: true
 
       - name: Test FSharp.Tests
-        if: always()
```

Comment:
Don't we want to continue if these tests fail?

Reply:
now it works exactly like that

Reply:
Why `continue-on-error: true` is added to `Test FSharp.Common.Tests` but not to `Test FSharp.Tests`? Does it affect both configurations somehow? Will next configurations (e.g. frontend tests) run properly if `Test FSharp.Tests` fails?